engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -42,12 +42,16 @@
                     <div class="g-block">
                         <div class="g-content">
                             <div class="g-array-item">
-                                {% if display.image.enabled and article.images.image_intro %}
-                                    <div class="g-array-item-image">
-                                        <a href="{{ article.route }}">
-                                            <img src="{{ url(article.images.image_intro) }}" {{ article.images.image_intro|imagesize|raw }} />
-                                        </a>
-                                    </div>
+                                {% if display.image.enabled %}
```

Comment:
Yes, if you see how it was before:

```
{% if display.image.enabled and article.images.image_intro %}
```

That was checking if the display image setting was enabled as well as making sure that the image_intro was set (with ` and article.images.image_intro `). In your change however, this check is gone and we need to have it for both `image_intro` and `image_fulltext`, based on whatever setting has been set.

Does it make sense?